docs/integrating-with-linters.md

```diff
@@ -0,0 +1,119 @@
+---
+id: integrating-with-linters
+title: Integrating with Linters
+---
+
+Prettier can be integrated into workflows with existing linting tools.
+This allows you to use Prettier for code formatting concerns, while letting your linter focus on code-quality concerns as outlined in our [comparison with linters](comparison.md).
+
+Whatever linting tool you wish to integrate with, the steps are broadly similar.
+First disable any existing formatting rules in your linter that may conflict with how Prettier wishes to format your code. Then you can either add an extension to your linting tool to format your file with Prettier - so that you only need a single command for format a file, or run your linter then Prettier as separate steps.
+
+All these instructions assume you have already installed `prettier` as a developer dependency.
```

Comment:
```suggestion
All these instructions assume you have already installed `prettier` as in your `devDependencies`.
```

I know it's not a real word, but it makes it immediately clear where to put it. I've always thought of it as "development dependencies", so it didn't immedialy click for me. What do you think?

Also, there are a couple more occurrences of "developer dependencies" below.

Reply:
Good call. You're right it is "development dependencies" and I just had a brainfart. `devDependencies` certainly offers clearer intent